swagger_django_generator/templates/django/utils.py

```diff
@@ -57,6 +57,14 @@ def wrapper(request, *args, **kwargs):
                         request.user = user
                         return view_func(request, *args, **kwargs)
 
+                    if "HTTP_X_API_KEY" in request.META:
+                        key = request.META["HTTP_X_API_KEY"]
+                        keys = set(os.getenv("ALLOWED_API_KEYS").split(","))
+                        if key in keys:
+                            return view_func(request, *args, **kwargs)
+                        else:
```

Comment:
You can drop the `else HttpResponse("Forbidden", status=403)` and let it fall through to the `return HttpResponse("Unauthorized", status=401)`. The 401 is actually correct. I incorrectly said 403 in the ticket.
For interest's sake:
* 403 is used when someone is authorised but do not have access
* 401 is used when someone is not authorised (which is the case here)